src/Simplex/Chat/Controller.hs

```diff
@@ -403,6 +403,7 @@ data ChatCommand
   | APIAddContact UserId IncognitoEnabled
   | AddContact IncognitoEnabled
   | APISetConnectionIncognito Int64 IncognitoEnabled
+  | APISetConnectionUserId Int64 UserId
```

Comment:
suggest adding a comment here to avoid some confusion in future, as I think it's the first time we're switching anything from user to user, and UserId is used in apis all the time to refer to current user (object's user), with exception of user manipulation apis.
```suggestion
  | APISetConnectionUserId Int64 UserId -- new user id to switch connection to
```